installer/upgrader/Migrations/V5/leaveLangString.yaml

```diff
@@ -0,0 +1,287 @@
+    unitId: no_leave_types_with_leave_balance
+  -
+    value: 'Partial Days'
+    unitId: partial_days
+  -
+    value: 'Assign Leave'
+    unitId: assign_leave
+  -
+    value: 'Start Day'
+    unitId: start_day
+  -
+    value: 'End Day'
+    unitId: end_day
+  -
+    value: 'Assign'
+    unitId: assign
+  -
+    value: 'Leave List'
+    unitId: leave_list
+  -
+    value: 'Show Leave with Status'
+    unitId: show_leave_with_status
+  -
+    value: 'My Leave Request Details'
+    unitId: my_leave_request_details
+  -
+    value: 'Leave Request Details'
+    unitId: leave_request_details
+  -
+    value: 'My Leave List'
+    unitId: my_leave_list
+  -
+    value: 'Show Leave with Status'
+    unitId: show_leave_with_status
+  -
+    value: 'Updating Entitlement'
+    unitId: updating_entitlement
+  -
+    value: 'Matching Employees'
+    unitId: matching_employees
+  -
+    value: 'The selected leave entitlement will be applied to the following employees.'
+    unitId: selected_leave_entitlement_applied_to_following_employees
+  -
+    value: 'Old Entitlement'
+    unitId: old_entitlement
+  -
+    value: 'New Entitlement'
+    unitId: new_entitlement
+  -
+    value: 'Situational Leave'
+    unitId: situational_leave
+  -
+    value: 'These leave will be excluded from reports unless there''s some activity. 
+    E.g. maternity leave, jury duty leave.'
+    unitId: entitlement_situational_description
+  -
+    value: 'Confirm Leave Assignment'
+    unitId: confirm_leave_assignment
+  -
+    value: 'Employee does not have sufficient leave balance for leave request.'
+    unitId: employee_does_not_have_enough_balance_for_leave_request
+  -
+    value: 'Click OK to confirm leave assignment.'
+    unitId: click_ok_to_confirm_leave_assignment
+  -
+    value: 'Existing Entitlement value {oldvalue} will be updated to {newvalue}'
+    unitId: entitlement_value_confirmation_message
+  -
+    value: 'Leave Balance'
+    unitId: leave_balance
+  -
+    value: 'Updating Entitlement'
+    unitId: updating_entitlement
+  -
+    value: 'Balance not sufficient'
+    unitId: balance_not_sufficient
+  -
+    value: 'Insufficient Leave Balance'
+    unitId: insufficient_leave_balance
+  -
+    value: 'Balance'
+    unitId: balance
+  -
+    value: 'Days'
+    unitId: days
+  -
+    value: 'Available Balance'
+    unitId: available_balance
+  -
+    value: 'Leave Balance Details'
+    unitId: leave_balance_details
+  -
+    value: 'As of Date'
+    unitId: as_of_date
+  -
+    value: 'Total Entitlement'
+    unitId: total_entitlement
+  -
+    value: 'Leave Status'
+    unitId: leave_status
+  -
+    value: 'Taken'
+    unitId: taken
+  -
+    value: 'Scheduled'
+    unitId: scheduled
+  -
+    value: 'Pending Approval'
+    unitId: pending_approval
+  -
+    value: '{amount, plural, =0{You are about to {action} no Leave Requests} one{You are about to {action} one Leave Request} other {You are about to {action} {#} Leave Requests}}'
```

Comment:
*

Reply:
I need to think a little bit on this since the action is supposed to be dynamic in these two strings and I've overlooked the conversion of that at the moment so will check and push that